e2e/playwright/actions/src/tests/create-folder-from-template.spec.ts

```diff
@@ -0,0 +1,418 @@
+/*!
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Alfresco Example Content Application
+ *
+ * This file is part of the Alfresco Example Content Application.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail. Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * The Alfresco Example Content Application is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The Alfresco Example Content Application is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * from Hyland Software. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { expect } from '@playwright/test';
+import {
+  AcaHeader,
+  ContentNodeSelectorDialog,
+  CreateFromTemplateDialogComponent,
+  DataTableComponent,
+  NodeContentTree,
+  NodesApi,
+  Utils,
+  errorStrings,
+  getUserState,
+  test
+} from '@alfresco/playwright-shared';
+
+test.use({ storageState: getUserState('hruser') });
+test.describe('Create folder from template', () => {
+  let selectFolderTemplateDialog: ContentNodeSelectorDialog;
+  let createFolderFromTemplateDialog: CreateFromTemplateDialogComponent;
+  let dataTable: DataTableComponent;
+  let toolbar: AcaHeader;
+  let randomFolderName: string;
+  let randomFolderTitle: string;
+  let randomFolderDescription: string;
+  let folderLink: string;
+  let folderId: string;
+  const selectDialogTitle = 'Select a folder template';
+  const dialogBreadcrumb = 'Space Templates';
+  const cancelButton = 'CANCEL';
+  const createButton = 'Create';
+  const nameLabel = 'Name *';
+  const titleLabel = 'Title';
+  const descriptionLabel = 'Description';
+  const emptyString = '';
+  const tabKeyString = 'Tab';
+  const dotString = '.';
+  const spaceString = '   ';
+  const commandKey = 'Meta';
+  const random = Utils.random();
+
+  const fileInRootFolder = `afile-in-root-${random}.txt`;
+  const folderInRootFolder = `afolder-in-root-${random}`;
+
+  const templateFolder1 = `aatemplate-folder1-${random}`;
```

Comment:
fix variable naming not to trigger linter

Reply:
done